CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 0.6.5 - 2023-12-18
+### Upgraded
+- 🏳️ Translations
+- 🐞 Bug fixing
```

Comment:
Would be nice to list the actual pull requests that were merged for the bug fixing ;)